pkg/workloads/cortex/downloader/download.py

```diff
@@ -23,20 +23,23 @@
 
 
 def start(args):
-    download = json.loads(base64.urlsafe_b64decode(args.download))
-    for download_arg in download:
+    download_config = json.loads(base64.urlsafe_b64decode(args.download))
+    for download_arg in download_config["download_args"]:
         from_path = download_arg["from"]
         to_path = download_arg["to"]
         item_name = download_arg.get("item_name", "")
```

Comment:
`item_name = download_arg.get("item_name")` then check `if item_name is not None:` unless defaulting to empty string is deliberate